packages/orbit-components/src/HorizontalScroll/index.tsx

```diff
@@ -37,6 +39,7 @@ const ArrowButton = ({
       )}
       onClick={onClick}
       type="button"
+      title={title}
```

Comment:
Why did you choose `title` attribute, not `aria-label`? From what I read, if the tooltip on the button is not needed, then the `aria-label` should be preferred over `title` (article [here](https://mauss.dev/posts/aria-label-vs-title-attribute)).

Reply:
I was doubting between the two until the last moment. I considered the title bringing more information when hovering it on top of what aria-label provides, that's the only reason why I chose it. I don't have a strong opinion between both, as you said it's a matter of deciding whether we need the tooltip included with the title or not.

Do you have a strong preference for area-label in this use-case?

Reply:
I would personally go in the `aria-label` way, we don't need tooltips on these buttons and many blog posts/websites say that `aria-label` is more supported by various types/models of screen readers. I would say it's more suitable if we want to cover a11y as much as possible.

Reply:
After reading [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#accessibility_concerns), I believe you're right, it's better to user aria labels, I'll adapt the PR.
Please, next time link the appropriate resources 😅, the article you shared doesn't look so reliable.

Thank you!

Reply:
Adjusted, thanks again! 👍